site/docs/java-api-quickstart.md

```diff
@@ -0,0 +1,141 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Java API Quickstart
```

Comment:
Can we rename this file in a separate commit? Then it would be easier to see changes.

Reply:
Am I correct that this file is not new, it is already in the documentation? We just renamed it?

Reply:
Am I correct that this file is not new, it is already in the documentation? We just renamed it?

Reply:
This is a copy of the original `api-quickstart` page with the code bits updated to use Java rather than Scala. I left the original page but changed the title on it to 'Spark API Quickstart' to differentiate it from the Java one here (: